include/cutlass/gemm/device/gemm_universal_adapter.h

```diff
@@ -402,7 +402,11 @@ class GemmUniversalAdapter<
         const auto sycl_block = syclcompat::dim3(block.x, block.y, block.z);
         const auto sycl_grid = syclcompat::dim3(grid.x, grid.y, grid.z);
 
+#if defined (SYCL_INTEL_TARGET)
```

Comment:
Since, for Nvidia the number is 32 and this one is the superset of the other one,can you set the sg in the device architecture parameters that we add use this parameters hereo for both  Nvidia and intel. In this case one of Def can be avoided.